CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+### 18.5.4 - [#784](https://github.com/openfisca/openfisca-france/pull/784)
+
+* Évolution du système socio-fiscal.
+* Périodes concernées : à partir du 01/01/2017
+* Zones impactées : ` openfisca_france/model/prestations/minima_sociaux/ass.py`.
```

Comment:
The normalized format doesn't include the `.py` extension, nor the `openfisca_france/model` prefix